code/modules/atmospherics/machinery/portable/canister.dm

```diff
@@ -689,6 +693,11 @@ GLOBAL_LIST_INIT(gas_id_to_canister, init_gas_id_to_canister())
 							var/msg = "[name]: [danger[name]] moles."
 							log_admin(msg)
 							message_admins(msg)
+					else
+						log_admin("[key_name(usr)] opened a canister that contains the following at [AREACOORD(src)]:")
+						for(var/name in gaseslog)
+							var/msg = "[name]: [gaseslog[name]] moles." //owo shrug
+							log_admin(msg)
```

Comment:
If someone opens a canister that contains o2 and plasma, I still want to know about the o2 right? Could you mesh this with the danger printouts somehow you think?

Reply:
that does make sense :o I'll look at it in a second